### PR TITLE
Patrick/upgrade cli version

### PR DESCRIPTION
**Pull request by pdougall1**
I was having some problems getting things set up on  my local machine.  Mainly I think newer Ember Cli versions have newer ES2015 transpiler and things weren't playing nice.  This upgrades to 1.13.13 which fixes the problem.  

Within the init of the new version of Cli it became obvious we can't use `ember-deploy` anymore, and are instead to use `ember-cli-deploy`.  That upgrade is important and should be a separate PR.

May need up upgrade your global Ember Cli version, I don't know http://ember-cli.com/user-guide/#setup

Also solves a couple more problems we were having :)

### CODE REVIEW

**Comment by gschorkopf** [comment]
Thanks for the PR. I'll look at it later today.

If you recall: what kind of error messages were you getting when setting up locally? And why did you "keep some changes" but not others in those commits?

This whole project needs a clean up, so thanks for getting started!

**Comment by pdougall1** [comment]
Sure, the error was something about `define is undefined` in the api_key adapter 
`import DS from 'ember-data';` was getting transpilied into `define('some path or something')` but `define` was apparently no longer, well, defined

When running `ember init`, the process re-writes a lot of files.  Many of those are things we want, and you see them in the commit.  Many are things we don't want, like deleting all the routes.  Potentially you can see what I didn't keep by running `ember init` and saying yes to everything.  All the changes will be what I didn't keep.

The updates in the package files (npm and bower) reflect updates to the packages that ember cli updated, nothing else was updated.

**Comment by gschorkopf** [comment]
Sorry, "Sure, the error was something about define is undefined in the api_key adapter 
import DS from 'ember-data'; was getting transpilied into define('some path or something') but define was apparently no longer, well, defined" is fairly confusing. How does this impact running anything locally?

**Comment by pdougall1** [comment]
`define is not defined` is an error that was getting thrown.  That error stopped the javascript process from loading the ember application, therefor nothing was able to run locally.

Sorry if that is confusing.  Essentially you had an old version of Cli that was using a transpiler that used the keyword`define` in it's module system.  That keyword (`define`) isn't understood in the current module system so it was throwing the error.  The `api_key` adapter is likely the first file that was getting transpiled, so that is where the error was getting thrown.

**Comment by gschorkopf** [comment]
Haha, I didn't expect the `define is not defined` part to be so literal. Thanks again for the PR!

File: bower.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-    "ember-data": "1.0.0-beta.15",
-    "ember-resolver": "~0.1.11",
-    "loader.js": "ember-cli/loader.js#1.0.1",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.0.2",
-    "ember-qunit": "0.2.8",
-    "ember-qunit-notifications": "0.0.7",
-    "qunit": "~1.17.1",
+    "ember": "1.13.11",
```
Comment (gschorkopf) [inline bower.json:4]: Does this update mean that most deprecations are gone? Are we ready for 2.0?

File: config/environment.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ module.exports = function(environment) {
   };
 
   if (environment === 'development') {
+    ENV.apiHost = 'http://localhost:3000'
```
Comment (gschorkopf) [inline config/environment.js:29]: :+1:

**Comment by gschorkopf** [comment]
So sorry it took so long to get around to this! Your changes broke a few things, so I made some changes of my own. There are also a bunch of deprecation warnings, which will be my next task.

Again, thanks for doing this PR! And at 6 months of long last... 🚢ing

**Pull request merged**